fleetspeak/client-win/fleetspeak.wxs

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?define UPGRADECODE="0D58409B-F48A-463A-80B8-654A69EC05F8"?>
+
+<!-- We want the installer to create Fleetspeak's log directory on
+     install. Wix requires us to have a static GUID for the directory. -->
+<?define LOGDIRGUID="0D58409B-F48A-463A-80B8-654A69EC05F8"?>
+
+<Wix
+    xmlns="http://schemas.microsoft.com/wix/2006/wi"
+    xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Product
+      Name="Fleetspeak"
+      Id="*"
+      UpgradeCode="$(var.UPGRADECODE)"
+      Language="1033"
+      Version="$(var.VERSION)"
+      Manufacturer="Fleetspeak">
+
+    <Package
+        Id="*"
+        Keywords="Installer"
+        Description="Installer for Fleetspeak version $(var.VERSION)"
+        InstallerVersion="200"
+        InstallScope="perMachine"
+        Languages="1033"
+        Compressed="yes" />
+
+    <!-- Allow rolling back to old Fleetspeak versions. -->
+    <!-- If an upgrade fails, roll back to existing version of Fleetspeak. -->
+    <MajorUpgrade AllowDowngrades="yes" Schedule="afterInstallInitialize" />
+
+    <Media Id="1" Cabinet="fleetspeak.cab" EmbedCab="yes" />
+
+    <!-- Installation directories. -->
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <!-- See http://wixtoolset.org/documentation/manual/v3/bundle/bundle_built_in_variables.html -->
+      <Directory Id="ProgramFiles64Folder">
+        <Directory Id="INSTALLDIR" Name="Fleetspeak" />
+      </Directory>
+
+      <!-- Custom directory where Fleetspeak will keep its logs. -->
+      <Directory Id="LOGDIR" />
+    </Directory>
+
+    <SetDirectory Id="LOGDIR" Value="[%SYSTEMROOT]\Temp\Fleetspeak" />
+
+    <DirectoryRef Id="INSTALLDIR">
+      <!-- It is recommended to have at most one file per component. -->
+      <!-- Note that service deletion does not occur on uninstall if we use
+           a fixed component-guid (so we generate a new one every time). -->
+      <Component Id="fleetspeak.exe" Guid="*">
+        <File
+            Id="fleetspeak.exe"
+            Name="fleetspeak.exe"
+            DiskId="1"
+            Source="$(var.FLEETSPEAK_EXECUTABLE)"
+            Checksum="yes"
+            KeyPath="yes" />
+        <!-- -stderrthreshold=3 turns off logging to stderr, which would
+             kill the Windows service. -->
+        <ServiceInstall
+            Arguments="-config_dir=HKEY_LOCAL_MACHINE\SOFTWARE\Fleetspeak -stderrthreshold=3 -log_dir=[%SYSTEMROOT]\Temp\Fleetspeak"
```

Comment:
Same here. Regex key root naming should match binary/service name.

Reply:
Done. Using FleetspeakClient (without space) name consistently.